docs/photoniq/vwrs/configure-vwrs-edgeworker.md

```diff
@@ -3,7 +3,7 @@ sidebar_position: 30
 title: Configure a VWRs EdgeWorker
 ---
 
-Import the `VirtualWaitingRoom` class from the virtualWaitingRoom.js file to configure a waiting room.
+Import the `VirtualWaitingRoom` class from the [virtualWaitingRoom.js file](NEED FILE) to configure a waiting room.
```

Comment:
@rahul-naik1 and @joubert-macrometa, what file is this referring to? Also, where does the user do this? Is there an Akamai page we can link to or something? Or should we say "Have your Akamai SE set this up for you" or something similar?

Reply:
In the EdgeWorker, import the 'VirtualWaitingRoom' class from the xxx Node Package Manager (NPM).  This integrates the VWRs with the EdgeWorker.

Reply:
Is XXX the actual name?

Reply:
Thanks! This is now just waiting on the updated API.